src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.hpp

```diff
@@ -225,19 +223,19 @@ class ShenandoahDirectCardMarkRememberedSet: public CHeapObj<mtGC> {
   size_t total_cards() const;
   size_t card_index_for_addr(HeapWord *p) const;
   HeapWord *addr_for_card_index(size_t card_index) const;
-  inline const CardValue* get_card_table_byte_map(bool write_table) const;
```

Comment:
Should these smaller, non-templated functions remain inline?

Reply:
ok, let me revert the specifier back.